portfolio/src/main/webapp/script.js

```diff
@@ -145,4 +145,161 @@ function dark() {
     document.getElementById('poem_1').style.backgroundColor = 'black';
     document.getElementById('poem_2').style.backgroundColor = 'black';
     document.getElementById('poem_3').style.backgroundColor = 'black';
-}
\ No newline at end of file
+}
+
+var amplitude = 70;
+var left_pos = 600;
+var top_pos = -600;
+var pi = Math.PI;
+var inc = pi / 30;
+var position_1=0;
+var position_2=0;
+var position_3=0;
+var position_4=0;
+var position_5=0;
+var position_6=0;
+var position_7=0;
+var position_8=0;
+var position_9=0;
+var position_10=0;
+
+function fly_1() {
```

Comment:
Can you find a way to write this code without writing ten `fly_*` functions with ten `position_*` variables? This is a good start to show that it's working. However your current approach also introduces a lot of duplicate code. I think you can write the code more succinctly by defining a function like this

`function fly(fact_index, angle, pos_left, pos_top, ...)`

This could be a good follow up change to this one.

Reply:
Yes, I was about to do that today, it's just that I found out about event listeners quite late and before that separated the functions while trying to make them work at the same time

Reply:
You can follow up in another change if you want. You can consider to start with just one or two list items and make it work before making it work for ten? That may require less effort to get things working at the beginning.